digestwriter/consumer.go

```diff
@@ -220,7 +220,12 @@ func (d *DigestConsumer) sendPayloadTrackerMessage(event *utils.PayloadTrackerEv
 	logger.Debugf("sending Payload Tracker message with status %s", event.Status)
 	if err := event.SendKafkaMessage(d.PayloadTracker); err != nil {
 		logger.Errorf("failed to send Payload Tracker message: %s", err.Error())
+		payloadTrackerError.Inc()
+		return
 	}
+
+	payloadTrackerMessageSent.Inc()
+	logger.Infof("successfully sent Payload Tracker message req=%v, status=%s", event.RequestID, event.Status)
```

Comment:
please log this on debug level as well, it's not that important message

let's try to keep single message for one succesfuly processed cluster in production logs